bufler.el

```diff
@@ -672,7 +672,11 @@ buffer's depth in the group tree."
                                        'face 'bufler-mode))))
     (concat (make-string (* 2 depth) ? )
             mode-annotation
-            (buffer-name buffer))))
+            (buffer-name buffer)
+            (propertize (if (and (buffer-file-name buffer)
```

Comment:
It seems that `buffer-modified-p` returns nil for non-file-backed buffers, so I think we don't need to test `buffer-file-name` as well.

Reply:
removed